include/boost/geometry/strategies/geographic/distance_cross_track.hpp

```diff
@@ -254,17 +273,22 @@ private :
                                                       spheroid).distance
                       << std::endl;
 #endif
-            if (inverse_distance_type::apply(lon3, lat3, pl_lon, pl_lat, spheroid).distance
-                < inverse_distance_type::apply(lon3, lat3, pr_lon, pr_lat, spheroid).distance)
+            CT const dist_l = inverse_distance_type::apply
+                (lon3, lat3, pl_lon, pl_lat, spheroid).distance;
+            CT const dist_r = inverse_distance_type::apply
+                (lon3, lat3, pr_lon, pr_lat, spheroid).distance;
```

Comment:
Accroding to our guidelines lines can have up to 100 characters. I don't know if this would fit into 100 characters but in general putting parts of the expression in multiple lines makes it harder to read.

Reply:
The official boost line length is 80 characters. I don't remember that we changed it.

I would actually be in favor of making it larger - but then maybe even more than 100 (for example 120). Years ago Bruno objected to make it larger than 80 characters, because he sometimes edits sources in a terminal. That might still be valid for some users. Also for showing a side-by-side diff the max line length is important.

But most probably for this one, we're just still restricted to Boost Guideline which is 80.
https://www.boost.org/development/requirements.html

Reply:
@barendgehrels from https://github.com/boostorg/geometry/wiki/Guidelines-for-Developers

> The preferred line length is 80 characters, with maximum length of 100.

It's like that since guidelines were added to the [Wiki in 2014](https://github.com/boostorg/geometry/wiki/Guidelines-for-Developers/9d366b7a52005b3b995e79501c07adeb5d0d75a5). And I'm pretty sure that we talked about this then.

I'm ok with increasing it to 120.

Isn't line length mainly defined by the size of the screen? Nowadays we have bigger screens or multiple screens. This also affects the size of the terminal as well as widths of diff columns.

Reply:
It's in the original version from the docs, since 2013:
https://github.com/boostorg/geometry/commit/10dd010ecc189276de1584187de847c4b4239e3c#diff-f7aee8e109b4ecd10dec03a3010cc6c110436140b709fa3632a5053c57d1bb84

Reply:
> According to our guidelines lines can have up to 100 characters. I don't know if this would fit into 100 characters but in general putting parts of the expression in multiple lines makes it harder to read.

It doesn't fit. 
Is 
```
CT const dist_l = 
    inverse_distance_type::apply(lon3, lat3, pl_lon, pl_lat, spheroid).distance;
CT const dist_r = 
    inverse_distance_type::apply(lon3, lat3, pr_lon, pr_lat, spheroid).distance;
```
preferable?

Reply:
> @barendgehrels from https://github.com/boostorg/geometry/wiki/Guidelines-for-Developers
> 
> > The preferred line length is 80 characters, with maximum length of 100.
> 
> It's like that since guidelines were added to the [Wiki in 2014](https://github.com/boostorg/geometry/wiki/Guidelines-for-Developers/9d366b7a52005b3b995e79501c07adeb5d0d75a5). And I'm pretty sure that we talked about this then.

:+1: thanks for the reminder!

So I'm already setting the right margin in the IDE to 80. Anyway: "preferred length is 80" but you may exceed it is actually not such a clear guideline... But anyway, I think we all do.

> 
> I'm ok with increasing it to 120.
> 
> Isn't line length mainly defined by the size of the screen? Nowadays we have bigger screens or multiple screens. This also affects the size of the terminal as well as widths of diff columns.

Sorry, I've to retract my own statement about 120, it's probably too much. Yes, the length is of course determined by the screen width - but also by the used font. I just did a `meld` (side by side view) in some adapted code and for me 107 is the personal limit. And of course for others the limit might be higher, but also smaller, all depending on personal settings.

So maybe better to just keep it as it is written there.